animeal/src/Flows/AttachPhoto/Model/AttachPhotoModel.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import Services
+import Amplify
+
+final class AttachPhotoModel: AttachPhotoModelProtocol{
+    // MARK: - Private properties
+    private let mapper: AttachPhotoModelMapperProtocol
+    
+    typealias Context = NetworkServiceHolder & DataStoreServiceHolder & UserProfileServiceHolder
+    private let context: Context
+    
+    // MARK: - DataStore properties
+    let feedingPointId: String
+    
+    private var feedingPoint: FeedingPoint?
+    
+    // MARK: - Initialization
+    init(
+        pointId: String,
+        mapper: AttachPhotoModelMapperProtocol = AttachPhotoModelMapper(),
+        context: Context = AppDelegate.shared.context
+    ) {
+        self.feedingPointId = pointId
+        self.mapper = mapper
+        self.context = context
+    }
+    
+    func fetchFeedingPoints(_ completion: ((AttachPhotoModel.PointContent) -> Void)?) {
+        context.networkService.query(request: .get(FeedingPoint.self, byId: feedingPointId)) { [weak self] result in
+            guard let self = self else { return }
+            switch result {
+            case .success(let point):
+                guard let point = point else {
+                    return
+                }
+                self.feedingPoint = point
+                Task {
+                    DispatchQueue.main.async {
```

Comment:
We have to use `@MainActor` instead of GCD in such cases. Will update all the places where `DispatchQueue.main.async` is used.

Reply:
However it's model we don't have to switch to the main queue here because of the fact that all methods in presenter are isolated with `@MainActor`.

Reply:
I will change later it later by myself.